po/de.po

```diff
@@ -4209,50 +4173,46 @@ msgid ""
 msgstr ""
 "Wie im [Rust RFC Book][1] erwähnt, werden diese bei der Verwendung nicht "
 "inliniert und haben einen tatsächlich zugeordneten Speicherort. Dies ist "
-"nützlich für unsicheren und eingebetteten Code, und die Variable lebt "
-"während der gesamten Programmausführung."
+"nützlich für unsicheren (unsafe) und eingebetteten (embedded) Code. Die "
+"Variable existiert während der gesamten Programmausführung."
 
 #: src/basic-syntax/static-and-const.md:44
-#, fuzzy
 msgid ""
 "We will look at mutating static data in the [chapter on Unsafe Rust](../"
 "unsafe.md)."
 msgstr ""
-"Wir werden uns im [Kapitel über Unsafe Rust](../unsafe.md) mit mutierenden "
-"statischen Daten befassen."
+"Wir werden uns im [Kapitel über unsicheres Rust](../unsafe.md) mit "
+"mutierenden statischen Daten befassen."
```

Comment:
I think the better translation here is "veränderbaren" instead of "mutierenden". The intention is not that the data mutates by itself, but that we are mutating the data. If you want to translate more directly, I think the following would also work: 

"Wir werden uns im Kapitel über unsicheres Rust mit dem Verändern von statischen Daten befassen."

Reply:
Makes sense, I don't care which version we use though, "veränderbaren" or "mit dem Verändern von" is indifferent to me.